src/components/layout/MenuBar.tsx

```diff
@@ -8,90 +8,101 @@ import ApartmentIcon from '@mui/icons-material/Apartment';
-      </Link>
-
-      {/* Companies */}
-      <Link className="m-auto cursor-pointer" to="/companies">
-        <ApartmentIcon fontSize="large" className="m-auto text-white" />
-      </Link>
+  // A reusable function to create NavLink classes dynamically
+  const linkClasses = ({ isActive }: { isActive: boolean }) =>
+    isActive
+      ? "m-auto text-cyan-800 transform scale-125 transition-transform duration-150"
+      : "m-auto text-white transform scale-100 transition-transform duration-150";
 
-      {/* Documents */}
-      <Link className="m-auto cursor-pointer" to="/documents">
-        <DescriptionIcon fontSize="large" className="m-auto text-white" />
-      </Link>
-
-      {/* Add New */}
-      <Link className="m-auto cursor-pointer" to="/add-new">
-        <PersonAddAlt1Icon fontSize="large" className="m-auto text-white" />
-      </Link>
-
-      {/* Account */}
-      <Link to="/userInformation"
-        className="m-auto mt-[10vh] text-[2.5vw] cursor-pointer rounded-full"
-        data-testid="update-user"
-      >
-        <PersonOutlineOutlinedIcon fontSize="large" className="m-auto w-1/5 rounded-full bg-cyan-500 text-white" />
-      </Link>
-
-      {/* Drop Down Shortcut Menu */}
-      <div className="flex flex-col items-center justify-center relative">
-        <button className="flex items-center justify-items-center" onClick={toggleDropDown}>
-          <AddCircleIcon fontSize="large" className="m-auto text-white justify-items-center" />
-        </button>
+  return (
+    <div className="flex flex-row h-screen">
+      <nav className="flex flex-col justify-items-center bg-gray-500 justify-evenly h-full mr-10 md:w-1/6 max-w-1/6">
         
-        <ul
-          className={`bg-cyan-600 m-4 shadow-md rounded-md transition-all duration-700 ease-in-out transform ${
-            isDropDownOpen ? 'scale-100 opacity-100 visible' : 'scale-95 opacity-0 invisible'
-          }`}
+        {/* Logo */}
+        <Link className="m-auto cursor-pointer" to="/">
+          <img className="m-auto w-1/2 cursor-pointer" src={logo} alt="Logo" />
+        </Link>
+
+        {/* Home */}
+        <NavLink className={linkClasses} to="/home">
+          <HomeIcon fontSize="large" />
+        </NavLink>
+
+        {/* Profile */}
+        <NavLink className={linkClasses} to="/profile">
+          <PersonIcon fontSize="large" />
+        </NavLink>
+
+        {/* Companies */}
+        <NavLink className={linkClasses} to="/companies">
+          <ApartmentIcon fontSize="large" />
+        </NavLink>
+
+        {/* Documents */}
+        <NavLink className={linkClasses} to="/documents">
+          <DescriptionIcon fontSize="large" />
+        </NavLink>
+
+        {/* Add New */}
+        <NavLink className={linkClasses} to="/add-new">
+          <PersonAddAlt1Icon fontSize="large" />
+        </NavLink>
+
+        {/* Account */}
+        <NavLink
+          to="/userInformation"
+          className={({ isActive }) =>
```

Comment:
why is this navlink using `isActive` while the others are using linkClasses? Because its currently linked to a component?